### PR TITLE
resolve issue721

### LINKED ISSUE #721: Unhelpful error message if component not installed
Need to throw a more useful error here: https://github.com/adaptlearning/adapt_framework/blob/master/src/core/js/views/adaptView.js#L54

### PR DESCRIPTION
**Pull request by davewallace**
Throw informative error for components added to config.json which aren't installed.
Fixes #721

### CODE REVIEW

File: src/core/js/views/adaptView.js [hunk 1]
```diff
@@ -48,10 +48,18 @@ define(function(require) {
...
                     var ChildView = this.constructor.childView || Adapt.componentStore[model.get("_component")];
-                    var $parentContainer = this.$(this.constructor.childContainer);
-                    model.set("_nthChild", nthChild);
-                    $parentContainer.append(new ChildView({model:model}).$el);
+                    if (ChildView) {
+                        var $parentContainer = this.$(this.constructor.childContainer);
+                        model.set("_nthChild", nthChild);
+                        $parentContainer.append(new ChildView({model:model}).$el);
+                    } else {
+                        throw Error('The component \'' +
```
Comment (oliverfoster) [inline src/core/js/views/adaptView.js]: Note: You don't neew the Error() syntax, can just throw a string > https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/throw

**Comment by oliverfoster** [comment]
+1

**Comment by hrajotia** [comment]
+1

**Comment by oliverfoster** [comment]
+1

**Comment by brian-learningpool** [comment]
+1

**Comment by hrajotia** [comment]
+1

**Pull request merged**